pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/signed-exchange.md

```diff
@@ -130,27 +130,37 @@ content-type: application/signed-exchange;v=b3
 ```
 
 [tip type="important"]
-The `v=b3` version string is the current version. This version will change. 
+The `v="1..100"` in the request will change. Do not match on this exact string; instead split on `;` and look for `google`.
```

Comment:
My advice, for folks running `amppackager`, is look only for presence of the `amp-cache-transform` header, and delegate value-parsing to `amppkg`. This is mainly so it can later be extended easily to support more AMP SXG caches (pending https://github.com/ampproject/amppackager/issues/156), and have publishers pick that change up automatically. Totally open to discussion on this point, e.g. if that's not feasible or advisable.

Also, this is under the "Testing" section. I'm not sure if there is a better section, so just a FYI there.

Reply:
Adjusted the text in https://github.com/ampproject/amp.dev/pull/2703/commits/22b87660af02130fdcbef9d5003bb09a07cac8b2 to suggest checking for the existence of `amp-cache-transform` only (i.e. ignore value). My original thought was that it's technically possible for a non-google packaging solution to exist (also why the `google` value exists at all) but yeah, this doc is probably not a helpful place to make this point...

Reply:
Ah I see; I wasn't considering that angle; yeah, agreed this is probably not the doc for that.

(There's the other angle of somebody running a non-amppkg solution but meant for Google; in that case, though, they'll want to pay attention to target _and_ version #s in that case.)